app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :post
+
+  after_save :update_comments_counter
+
+  def update_comments_counter
+    post.increment!(:comments_counter)
+  end
```

Comment:
[OPTIONAL]
- I would recommend making the `update_comments_counter` private method since we are calling this method only from inside the class and with this it is more secure.